### ISSUE TITLE
Script example documentation incorrect

### ISSUE THREAD

**Issue by jrwrigh** [qtile/qtile#2475]
# Issue description

[Example script in documentation](https://testqtiledocs.readthedocs.io/en/latest/manual/commands/scripting.html) does not work:

```py
from libqtile.command import Client
c = Client()
print c.screen.info()["index"]
```

`Client` does not exist in `libqtile.command`. The only class named `Client` is within the `ipc` module, but that cannot initialize without arguments as done in the example. I'm new to qtile, so I have no idea what it should be.

# Qtile version

0.17.0

**Comment by Fabien-Bernard** [comment]
Your link looks strange. Seems to be very old qtile. 
See https://docs.qtile.org/en/latest/manual/commands/scripting.html

**Comment by jrwrigh** [comment]
Ok, thanks for the updated link. Not sure how I got the old one, but looking at it I realize that it's written in Python 2, so definitely not right.

But the script in the new one also doesn't work for me.
```py
from libqtile.command.client import CommandClient
c = CommandClient()
print(c.screen.info()["index"])
```
exits with `AttributeError: 'CommandClient' object has no attribute 'screen'`. Running `dir(c)` doesn't show anything useful since nearly all methods are called via `__getattr__`.

**Comment by Fabien-Bernard** [comment]
I didn't check the example before sending the link, sorry for that, I thought It was only a documentation version problem.

To be more helpfull from the height of my current knowledge, when I play with this, it helps me to have two terminals open, one with my python interpreter, one with the qtile shell. Then when I write my script, I can display informations at each node while navigating the graph.

To correct the example of the doc, I see 2 possibilities :

- replace `CommandClient` by `InteractiveCommandClient`,

- Or, to stick with `CommandClient`, replace the print command with something like `print(c.navigate("screen", None).call("info")()["index"])`

**Comment by jrwrigh** [comment]
I can confirm that either of the solutions you proposed works. I can submit a PR, but I'm not sure as to which solution is more desirable. `InteractiveCommandClient` is used more often in the documentation, but [documentation specifies](http://docs.qtile.org/en/latest/manual/commands/index.html#introduction-object-graph) that the `CommandClient` is more precise, which  context of scripting (5th paragraph down)

Also, the `navigate()` method is not specified in the documentation. Looks like it takes the place of `select()`? (which is [discussed in the documentation](http://docs.qtile.org/en/latest/manual/commands/index.html?highlight=navigate#digging-deeper-command-objects))

**Comment by elParaguayo** [comment]
Personally, I prefer the `InteractiveCommandClient` as it's a bit easier to understand what's going on i.e. `c.screen.info()["index"]` is preferable to `c.navigate("screen", None).call("info")()["index"]`.

**Comment by elParaguayo** [comment]
Closed via #2844.
